localization/english/bookmark/bookmark_603_l_english.yml

```diff
@@ -9,19 +9,19 @@
 
  #Daelin Proudmoore
  daelin_character:0 "Admiral Daelin"
- daelin_character_desc:0 "Patriarch of one of Kul Tiras's great houses, lord of Tiragarde and commander of the fleet, Daelin Proudmoore was elected by his peers as Lord-Admiral, ruler of Kul Tiras. Under his leadership, the merchant kingdom became the strongest naval power in the world, and pledged its fleet to the Alliance's service. As Grand Admiral of the Alliance, Daelin Proudmoore brought victory against the Horde during the Second War, during which his son and heir Derek died to orcish hands, and remained one of the Alliance's strongest supporter after its end. Even if the sea protects his kingdom from threats, being epidemic or orc agression, the admiral remains vigilant and knows that his fleet will be vital to the Alliance in case of a new war, he also knows that whoever hold the sea...holds the world. What destiny awaits Daelin and Kul Tiras ? Being the Alliance's new base of power ? To lead humanity oversea in search for new lands to colonize ? To be washed over by the tides ?"
+ daelin_character_desc:0 "Patriarch of one of Kul Tiras's great houses, lord of Tiragarde, and commander of the fleet, Daelin Proudmoore was elected by his peers as Lord-Admiral, ruler of Kul Tiras. Under his leadership, the merchant kingdom became the strongest naval power in the world and pledged its fleet to the Alliance's service. As Grand Admiral of the Alliance, Daelin Proudmoore brought victory against the Horde during the Second War, during which his son and heir Derek died to orcish hands, and remained one of the Alliance's strongest supporters after its end. Even if the sea protects his kingdom from threats, being epidemic or orc aggression, the admiral remains vigilant and knows that his fleet will be vital to the Alliance in case of a new war, he also knows that whoever holds the sea...holds the world. What destiny awaits Daelin and Kul Tiras? Being the Alliance's new base of power? To lead humanity oversea in search of new lands to colonize? To be washed over by the tides ?"
```

Comment:
```suggestion
 daelin_character_desc:0 "Patriarch of one of Kul Tiras's great houses, lord of Tiragarde, and commander of the fleet, Daelin Proudmoore was elected by his peers as Lord-Admiral, ruler of Kul Tiras. Under his leadership, the merchant kingdom became the strongest naval power in the world and pledged its fleet to the Alliance's service. As Grand Admiral of the Alliance, Daelin Proudmoore brought victory against the Horde during the Second War, during which his son and heir Derek died to orcish hands, and remained one of the Alliance's strongest supporters after its end. Even if the sea protects his kingdom from threats, being epidemic or orc aggression, the admiral remains vigilant and knows that his fleet will be vital to the Alliance in case of a new war, he also knows that whoever holds the sea... holds the world. What destiny awaits Daelin and Kul Tiras? Being the Alliance's new base of power? To lead humanity oversea in search of new lands to colonize? To be washed away by the tides?"
```